tests/fixtures/bar-foos-bar-route.js

```diff
@@ -0,0 +1,6 @@
+Router.map(function() {
```

Comment:
@nathanhammond I think removing multiple instances of the specified route should be handled as a separate bug. It changes the structure of the `_remove` function as a whole, as well as the return type of `hasRoute`. I would consider it an incompatible change if `hasRoute` is used outside of the micro-library because it will have to change to return an array of `path`s rather than a single.

I'll be glad to work on it. I just think it should be handled separately. And cleaning up duplicate routes that aren't part of the removal target is a separate behavior altogether.